ifrs17/Constants/Validations.ipynb

```diff
@@ -202,16 +105,193 @@
         {
             "cell_type": "code",
             "source": [
-                "public static string Get (Warning w, params string[] s) => (w, s.Length) switch {",
-                "\n    // Import",
-                "\n    (Warning.ActiveDataNodeWithCashflowBOPI , 1) => $\"Cash flow with AoC Type: {AocTypes.BOP} and Novelty: {Novelties.I} for Group of Contract {s[0]} is not allowed because previous period data are available.\",",
-                "\n    (Warning.VariablesAlreadyImported       , 0) => $\"The import of the current file does not contain any new data. Hence, no data will be saved or calculations will be performed.\",",
-                "\n    (Warning.MandatoryAocStepMissing        , 3) => $\"The AoC step ({s[0]}, {s[1]}) is not imported for ({s[2]}).\",",
-                "\n    (Warning.ScenarioReCalculations         , 1) => $\"The present Best Estimate import makes the result of dependent Scenarios out of date. Hence, the following Scenarios are re-calculated: {s[0]}.\", ",
-                "\n    // Default",
-                "\n    (Warning.Generic                        , _) => $\"{s[0]}\",",
-                "\n    (_                                      , _) => $\"Warning not found.\"",
-                "\n};"
+                "public class Error : ValidationBase{",
+                "\n",
+                "\n    protected const string DefaultMessage = \"Error not found.\";",
```

Comment:
Same here: remove variable and move message in the switch.

Reply:
Same as in the comment before: it is used more than once.